src/zino/snmp.py

```diff
@@ -125,15 +125,15 @@ async def get(self, *oid: str) -> MibObject:
             )
         except PysnmpMibNotFoundError as error:
             raise MibNotFoundError(error)
-        self._raise_errors(error_indication, error_status, error_index, query)
+        self._raise_errors(error_indication, error_status, error_index, var_binds)
         return self._object_type_to_mib_object(var_binds[0])
 
     def _raise_errors(
         self,
         error_indication: Union[str, errind.ErrorIndication],
         error_status: str,
         error_index: int,
-        *query: ObjectType,
+        var_binds: Sequence[ObjectType],
```

Comment:
It would work, but the `getCmd` commands and such always returns a list, and this function is meant to process that. 
`error_index` points to something in `var_binds`, so taking a Sequence directly instead of *args felt more appropriate to me.

`Optional` is a good point, that might be applicable for all the arguments